### PR TITLE
fix: Nextjs Component Testing Example

### LINKED ISSUE #16054: @cypress/react recipes document nextjs example broken link
In [this file](https://github.com/cypress-io/cypress/blob/develop/npm/react/docs/recipes.md#nextjs) the link for the `nextjs` example is broken.

Is there a way to update it eventually?

I am looking for a working example of component tests and can't find it anywhere.

Cheers

### PR DESCRIPTION
**Pull request by itstheandre**
Added missing dependencies for users to be able to look at the dependencies and run the tests on their machines.

This code shouldn't affect anything code related on `cypress` package, is more for documentation's sake

Closes #16054

### CODE REVIEW

**Review by lmiller1990 — APPROVED** [review]

File: npm/react/examples/nextjs/package.json [hunk 1]
```diff
@@ -19,7 +19,10 @@
     "@zeit/next-sass": "^1.0.1",
     "check-code-coverage": "1.9.2",
     "cypress-circleci-reporter": "0.2.0",
-    "next": "10.1.2"
+    "next": "10.1.2",
+    "@cypress/webpack-dev-server": "^1.1.3"
```
Comment (lmiller1990) [inline npm/react/examples/nextjs/package.json:23]: Missing a comma. so npm install is failing on CI. Can you add comma?

**Review by lmiller1990 — CHANGES_REQUESTED** [review]
Cool, just one thing - CI is failing since the package.json is now invalid since it is missing a comma.

File: npm/react/examples/nextjs/package.json (hunk 1 above)
Comment (itstheandre) [reply npm/react/examples/nextjs/package.json:23]: Ah damn, of course

Comment (itstheandre) [reply npm/react/examples/nextjs/package.json]: Fixed @lmiller1990 💪

**Review by lmiller1990 — DISMISSED** [review]

**Comment by lmiller1990** [comment]
It looks like this *does* break a lot of things, due to how all the example projects use the local version of the webpack-dev-server, not one from npm. This is intentional; it helps ensure regressions are not introduced. Specifying a specific version in `package.json` breaks things. Sorry, I should have looked into this in more detail before I gave you the green light to make the PR.

I think we probably will need more complete project examples, but those will need to live in another repo (possibly something people can clone, including instructions on setting it up in your own project).

**Comment by itstheandre** [comment]
@lmiller1990 so you think iits better to either write some documentation, or eventually just create a `template` repository for people to just clone from there?

Maybe using `degit` or somethiing like that

**Comment by lmiller1990** [comment]
Probably some documentation for now - in the mid-term we are working on template so you can just do `yarn create cypress-tests --next` and it'll  scaffold it for you.

I'll work on some better docs once [this is fixed and merged](https://github.com/cypress-io/cypress/pull/16078), since at the moment the dependencies are a bit messy.

**Comment by itstheandre** [comment]
That is exciting!

By the way, with this component testing is it possible to merge both e2e and component testing coverage?

**Comment by lmiller1990** [comment]
At the moment the runners are separate, but moving forward the goal is to unify everything. I cannot tell you exactly when/how this will be happening, though.

**Pull request closed without merging**